CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ## 🔧 Tech
 
 * Update cozy-ui to 62.1.4 and material-ui to 4.12.3
+* Change Sentry url + add instance in tags context
```

Comment:
I think this change is a feature, a fix or a refactor 🤔 . This change may have impacted and should not be considered as a simple chore. What is your opinion @JF-Cozy ?

Reply:
Il n'y a pas de modif pour le user, c'est pour ça que j'ai exclu le feat (et ce n'est ni un fix ni un refactor).

après relecture le chore c'est `no production code change`... allez va pour feat, avançons :)